experimental/dds/tree2/src/core/tree/detachedFieldIndexTypes.ts

```diff
@@ -0,0 +1,15 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { NestedMap } from "../../util";
+import { ForestRootId } from "./detachedFieldIndex";
+
+export type Major = string | number | undefined;
+export type Minor = number;
+
+export interface DetachedFieldSummaryType {
```

Comment:
```suggestion
export interface DetachedFieldSummaryData {
```
is a bit more consistent with EditManager's naming conventions. obviously suggestion will require renames elsewhere  so can't be applied as-is :)

Reply:
Updated interface name :)